command-expansion-server/sql/commanding/tables/activity_instance_commands.sql

```diff
@@ -1,5 +1,6 @@
 create table activity_instance_commands (
   activity_instance_id integer not null,
+  -- TODO: Change this to be the actual simulated instance id (span_id?)
```

Comment:
Is this something we've made a decision on?

Side note: personally I'm okay with the column name being something that makes sense in the command expansion context, and have it refer to the `span` table's id.

Reply:
In either case, we need it to point to the activity instance

Reply:
Lets keep `activity_instance_id` for now as `span` might be renamed. Also, the user will be going through the Hasura API and they will understand `activity_instance_id`  better than `span_id` when they do their query.